### ISSUE TITLE
Error installing dependencies using pacman on Arch linux

### ISSUE THREAD

**Issue by fxdgear** [MycroftAI/mycroft-core#1967]
Fresh clone of master today (1/22/19)
Ran install script and got the following error
```
Interactive portion complete, now installing dependencies...

Installing packages...
warning: git-2.20.1-1 is up to date -- skipping
warning: python-3.7.2-3 is up to date -- skipping
warning: python-pip-18.1-1 is up to date -- skipping
warning: python-setuptools-1:40.6.3-1 is up to date -- skipping
warning: libffi-3.2.1-3 is up to date -- skipping
warning: swig-3.0.12-2 is up to date -- skipping
warning: flac-1.3.2-2 is up to date -- skipping
warning: curl-7.63.0-4 is up to date -- skipping
warning: icu-63.1-2 is up to date -- skipping
warning: libjpeg-turbo-2.0.1-1 is up to date -- skipping
:: There are 26 members in group base-devel:
:: Repository core
   1) autoconf  2) automake  3) binutils  4) bison  5) fakeroot  6) file
   7) findutils  8) flex  9) gawk  10) gcc  11) gettext  12) grep  13) groff
   14) gzip  15) libtool  16) m4  17) make  18) pacman  19) patch  20) pkgconf
   21) sed  22) sudo  23) systemd  24) texinfo  25) util-linux  26) which

Enter a selection (default=all): 
warning: autoconf-2.69-5 is up to date -- skipping
warning: automake-1.16.1-1 is up to date -- skipping
warning: binutils-2.31.1-4 is up to date -- skipping
warning: bison-3.2-1 is up to date -- skipping
warning: fakeroot-1.23-1 is up to date -- skipping
warning: file-5.35-1 is up to date -- skipping
warning: findutils-4.6.0-4 is up to date -- skipping
warning: flex-2.6.4-2 is up to date -- skipping
warning: gawk-4.2.1-2 is up to date -- skipping
warning: gcc-8.2.1+20181127-1 is up to date -- skipping
warning: gettext-0.19.8.1-3 is up to date -- skipping
warning: grep-3.3-1 is up to date -- skipping
warning: groff-1.22.3-8 is up to date -- skipping
warning: gzip-1.10-1 is up to date -- skipping
warning: libtool-2.4.6+42+gb88cebd5-2 is up to date -- skipping
warning: m4-1.4.18-2 is up to date -- skipping
warning: make-4.2.1-3 is up to date -- skipping
warning: pacman-5.1.2-2 is up to date -- skipping
warning: patch-2.7.6-7 is up to date -- skipping
warning: pkgconf-1.6.0-1 is up to date -- skipping
warning: sed-4.7-1 is up to date -- skipping
warning: sudo-1.8.27-1 is up to date -- skipping
warning: systemd-240.34-3 is up to date -- skipping
warning: texinfo-6.5-2 is up to date -- skipping
warning: util-linux-2.33.1-2 is up to date -- skipping
warning: which-2.21-3 is up to date -- skipping
warning: jq-1.6-1 is up to date -- skipping
warning: pulseaudio-12.2-2 is up to date -- skipping
warning: pulseaudio-alsa-2-4 is up to date -- skipping
resolving dependencies...
looking for conflicting packages...

Packages (14) python-stevedore-1.30.0-1  python-virtualenv-clone-0.5.0-1
              python2-appdirs-1.4.3-2  python2-packaging-18.0-1
              python2-pyparsing-2.3.1-1  python2-setuptools-1:40.6.3-1
              python2-six-1.12.0-1  python2-virtualenv-16.0.0-1  mpg123-1.25.10-1
              portaudio-190600_20161030-2  python-gobject-3.30.4-1
              python-virtualenv-16.0.0-1  python-virtualenvwrapper-4.8.2-2
              screen-4.6.2-1

Total Download Size:    5.04 MiB
Total Installed Size:  10.36 MiB

:: Proceed with installation? [Y/n] 
:: Retrieving packages...
 python-virtualenv-1...  1782.3 KiB  1013K/s 00:02 [#########################] 100%
 python-gobject-3.30...   224.7 KiB  2.13M/s 00:00 [#########################] 100%
 python2-appdirs-1.4...    13.3 KiB  4.34M/s 00:00 [#########################] 100%
 python2-pyparsing-2...    89.7 KiB  2.92M/s 00:00 [#########################] 100%
 python2-six-1.12.0-...    19.1 KiB  3.11M/s 00:00 [#########################] 100%
 python2-packaging-1...    30.5 KiB  4.97M/s 00:00 [#########################] 100%
 python2-setuptools-...   274.5 KiB  2.23M/s 00:00 [#########################] 100%
 python2-virtualenv-...  1782.5 KiB  2.43M/s 00:01 [#########################] 100%
 mpg123-1.25.10-1-x86_64  255.8 KiB  2.78M/s 00:00 [#########################] 100%
 screen-4.6.2-1-x86_64    514.5 KiB  2.75M/s 00:00 [#########################] 100%
 python-virtualenv-c...    11.3 KiB  0.00B/s 00:00 [#########################] 100%
 python-stevedore-1....    43.1 KiB  2.63M/s 00:00 [#########################] 100%
 python-virtualenvwr...    25.3 KiB  2.47M/s 00:00 [#########################] 100%
 portaudio-190600_20...    94.0 KiB  2.78M/s 00:00 [#########################] 100%
(14/14) checking keys in keyring                   [#########################] 100%
(14/14) checking package integrity                 [#########################] 100%
(14/14) loading package files                      [#########################] 100%
(14/14) checking for file conflicts                [#########################] 100%
error: failed to commit transaction (conflicting files)
python-virtualenv: /usr/bin/virtualenv exists in filesystem
python2-six: /usr/lib/python2.7/site-packages/six.py exists in filesystem
python2-six: /usr/lib/python2.7/site-packages/six.pyc exists in filesystem
python2-setuptools: /usr/bin/easy_install-2.7 exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/easy_install.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/easy_install.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/pkg_resources/__init__.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/pkg_resources/__init__.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/pkg_resources/py31compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/pkg_resources/py31compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/__init__.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/__init__.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/archive_util.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/archive_util.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/build_meta.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/build_meta.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/__init__.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/__init__.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/alias.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/alias.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_egg.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_egg.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_rpm.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_rpm.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_wininst.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/bdist_wininst.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_clib.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_clib.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_ext.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_ext.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_py.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/build_py.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/develop.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/develop.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/dist_info.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/dist_info.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/easy_install.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/easy_install.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/egg_info.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/egg_info.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_egg_info.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_egg_info.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_lib.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_lib.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_scripts.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/install_scripts.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/py36compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/py36compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/register.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/register.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/rotate.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/rotate.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/saveopts.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/saveopts.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/sdist.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/sdist.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/setopt.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/setopt.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/test.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/test.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/upload.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/upload.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/upload_docs.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/command/upload_docs.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/config.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/config.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/dep_util.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/dep_util.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/depends.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/depends.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/dist.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/dist.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/extension.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/extension.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/glibc.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/glibc.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/glob.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/glob.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/launch.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/launch.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/lib2to3_ex.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/lib2to3_ex.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/monkey.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/monkey.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/msvc.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/msvc.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/namespaces.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/namespaces.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/package_index.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/package_index.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/pep425tags.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/pep425tags.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py27compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py27compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py31compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py31compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py33compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py33compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py36compat.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/py36compat.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/sandbox.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/sandbox.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/script (dev).tmpl exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/script.tmpl exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/site-patch.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/site-patch.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/ssl_support.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/ssl_support.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/unicode_utils.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/unicode_utils.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/version.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/version.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/wheel.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/wheel.pyc exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/windows_support.py exists in filesystem
python2-setuptools: /usr/lib/python2.7/site-packages/setuptools/windows_support.pyc exists in filesystem
python-virtualenv-clone: /usr/bin/virtualenv-clone exists in filesystem
python-virtualenvwrapper: /usr/bin/virtualenvwrapper.sh exists in filesystem
python-virtualenvwrapper: /usr/bin/virtualenvwrapper_lazy.sh exists in filesystem
Errors occurred, no packages were upgraded.
```

This is mostly a pacman related issue I realize. BUT I feel like the installer should be able to navigate this.

**Comment by forslund** [comment]
Hi, thanks for the report.

No one on the team is using Arch so we're kind of relying on the community for keeping Arch up to date. I do however see that there are a bunch of outdated packages that aren't really used.

Can you try to remove the following packages from L263 in dev_setup.sh:

python-virtualenv python-gobject python-virtualenvwrapper

And see if that resolves the issue?

**Comment by adocampo** [comment]
It's complaining about those files already existing in the filesystem and the installer cannot overwrite them. I ususally do a dirty hack which is manually deleting the conflicting files.

**Comment by FruityWelsh** [comment]
So I did not get those errors on my manjaro (arch based) install (not that it really went smooth). Instead I get errors related to venv . I manged to get the core running by changing dev_setup.sh to run venv minus the "--without-pip" option and removing the rest of that function. 

Now the issue is msm doesn't work and skill fail to install.

**Comment by adocampo** [comment]
Just ran `dev_setup.sh`on arch and it worked as expected. On my workplace it worked last week on manjaro. Could you try on another computer/vm with manjaro to see if it fails so see if is something related to your computer?

**Comment by FruityWelsh** [comment]
So I ran Virt manager and created a manjaro VM (using the manjaro install iso), and installed it to disk.

This ran into other issues, during install and I didn't create a share drive to output the logs. Will try again, and now that I am thinking about it, just upload the output to github...

On my main machine I deleted /opt/mycroft, ${HOME}/mycroft, /tmp/mycroft and the mycroft-core dirs.
cloned a fresh git and ran bash dev_setup again
Mycroft is now working!
